src/test/resources/madalinapopescu/groups.xml

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
+
+<suite name="SuitaTestNG" verbose="1" >
+
+    <test name="CuraHealthcareLoghinTests">
+        <groups>
+            <run>
+                <exclude name="login"  />
+                <include name="redirect"  />
+            </run>
+        </groups>
+        <classes>
+            <class name="madalinapopescu.tests.tema7.CuraHealthcareLoghinTests"/>
+        </classes>
+    </test>
```

Comment:
You're missing closing suite tag - </suite> at the end of the file. Please add </suite> after </test> line. Thanks!

Reply:
After </test> please add </suite>, thanks!

Reply:
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >

<suite name="SuitaTestNG" verbose="1" >

    <test name="CuraHealthcareLoghinTests">
        <classes>
            <class name="madalinapopescu.tests.tema7.LoghinTestsCuraHealthcare"/>
            <class name="madalinapopescu.tests.tema7.CuraHealthcareMakeAppointmentTest"/>
        </classes>
    </test>
</suite>

Reply:
<suite name="SuitaTestNG" verbose="1" >

    <test name="CuraHealthcareLoghinTests">
        <classes>
            <class name="madalinapopescu.tests.tema7.LoghinTestsCuraHealthcare"/>
            <class name="madalinapopescu.tests.tema7.CuraHealthcareMakeAppointmentTest"/>
        </classes>
    </test>
</suite>

Reply:
La mine apare si suita inchisa

Reply:
Gata, acum am pus-o si in grupuri.... m-am prins mai greu :D

Reply:
great! thanks!